parsers/verilog.c

```diff
@@ -1630,6 +1633,18 @@ static int tagNameList (tokenInfo* token, int c)
 			if (c == '=')
 				c = skipExpression (c);
 		}
+		else if (c == '(' || c == '[') // should be instance
+		{
+			c = skipDimension(c);
+			c = skipPastMatch ("()");
+			c = skipWhite (c);
```

Comment:
I've sent a pull-request #2718 for this.  I am waiting for the approval by @masatake.
If it is merged earlier than this pull-request, remove this line.

Again add a test case which fails by this issue.  (Add a space in ");" at the end of one of module instantiations.)